config.rb

```diff
@@ -179,8 +137,33 @@ def sponsors_for level
     sponsors.select {|s| s.data.level == level}
   end
 
-  def crevalle
-    sponsors.detect { |s| s.data.level == 'crevalle' }
+  def get_sponsor_by_id id
+    data.sponsors.detect {|slug, data| data.id == id}.last
```

Comment:
Documenting this better. The reason is because this would actually return a two element array where the first element was the slug and the second element was the data hash (as you can see in the iteration declaration). That's just the way Middleman works. I wanted this function to only return the hash of data, so I used `#last`. But now I've reversed that decision and I'm homogenizing all these functions to be called `find_*` and return the `[slug, data]` array.